soc/arm64/xenvm/Kconfig.soc

```diff
@@ -5,5 +5,6 @@ config SOC_XENVM
 	bool "Xen virtual machine on aarch64"
 	select ARM64
 	select ARM_ARCH_TIMER
-	select GIC_V2
+	select GIC_V3 if XENVM_USE_GIC_V3
+	select GIC_V2 if !XENVM_USE_GIC_V3
```

Comment:
Why cannot this be set by defconfig only?

Reply:
defconfig can only set visible symbol GIC_V{3,2} is an invisible symbol